CRM/Oapproviderlistapp/Form/Experience.php

```diff
@@ -62,16 +62,16 @@ public function formRule($fields, $files, $self) {
         }
         $error = '';
         if (empty($fields[$fieldName])) {
-          if (strstr($fieldName, 'custom_36')) {
-            $error .= E::ts('Total number of hours') . ':<br />' . E::ts('Applicants are required to have at least 3,000 hours post-certification/registration experience.');
-          }
-          if (strstr($fieldName, 'custom_37')) {
-            $error = E::ts('Approximate number of hours that involved supervisory duties') . ':<br />' . E::ts('Applicants are required to have a minimum of 1,500 post-certification hours involving supervisory duties.');
-          }
           $error .= E::ts('All fields in Employment History are required.');
           $errors['_qf_default'] = $error;
           CRM_Core_Session::setStatus("", $error, "alert");
         }
+        elseif (strstr($fieldName, 'custom_36') && $fields[$fieldName] < 3000) {
+          $errors['_qf_default'] .= sprintf('<strong>%s</strong>: <br /> %s', E::ts('Total number of hours'), E::ts('Applicants are required to have at least 3,000 hours post-certification/registration experience.'));
```

Comment:
@monishdeb is there a reason why you are using _qf_default rather than the field name here?

Reply:
@seamuslee001 there is a core bug in using the multi-record custom group, which is dynamically rendered in the experience form via AJAX. So when I add any validation rule to fields element(s) it doesn't throw error message against the field but the forms react to the default  '_qf_default' submit button index and thus all the error messages are collected and shown in an alert message.  I tried to fix this core bug but seems like rendering mult-record custom group block in a public page is not yet supported in core and has various issues and not validatiing the field elements is one of them.